plugins/woocommerce/includes/wc-template-functions.php

```diff
@@ -363,8 +363,9 @@ function wc_body_class( $classes ) {
  * @since 3.4.0
  */
 function wc_no_js() {
+	$type_attr = current_theme_supports( 'html5', 'script' ) ? '' : " type='text/javascript'";
 	?>
-	<script type="text/javascript">
+	<script<?php echo $type_attr; ?>>
```

Comment:
Looks like we need a tweak to satisfy the linter. There are a couple of approaches we might take. My first suggestion (since the reality is we're outputting a safe/known string constant):

```php
	<script<?php echo $type_attr; // phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped ?>>
```

Or else we could do:

```php
	$type_attr = current_theme_supports( 'html5', 'script' ) ? '' : 'text/javascript';
	?>
	<script<?php echo empty( $type_attr ) ? '' : ' type="' . esc_attr( $type_attr ) . '"'; ?>>
```

Reply:
Added phpcs ignore comment for unescaped output